pkg/uefi/patch/000.patch

```diff
@@ -1,25 +1 @@
---- BaseTools/Source/C/Include/AArch64/ProcessorBind.h.orig
-+++ BaseTools/Source/C/Include/AArch64/ProcessorBind.h
-@@ -61,7 +61,9 @@
-   typedef char                CHAR8;
-   typedef signed char         INT8;
- 
-+#ifndef UINT8_MAX
-   #define UINT8_MAX 0xff
-+#endif
- #endif
- 
- ///
---- OvmfPkg/PlatformPei/Xen.c.orig
-+++ OvmfPkg/PlatformPei/Xen.c
-@@ -137,6 +137,10 @@
-       mXen = TRUE;
-       return TRUE;
-     }
-+    if (!AsciiStrCmp ((CHAR8 *) Signature, "ZenZenZenZen")) {
-+      mXen = TRUE;
-+      return TRUE;
-+    }
-   }
- 
-   mXenLeaf = 0;
+diff --git OvmfPkg/PlatformPei/Xen.c OvmfPkg/PlatformPei/Xen.c
index 104922c67e..f1eae10d2d 100644
--- OvmfPkg/PlatformPei/Xen.c
+++ OvmfPkg/PlatformPei/Xen.c
@@ -137,6 +137,10 @@ XenDetect (
       mXen = TRUE;
       return TRUE;
     }
+    if (!AsciiStrCmp ((CHAR8 *) Signature, "ZenZenZenZen")) {
+      mXen = TRUE;
+      return TRUE;
+    }
   }
 
   mXenLeaf = 0;

```

Comment:
This doesn't seem right with all the ^M characters -- is this just a formatting issue of some kind?

Reply:
Sorry, @rvs, it is my mistake. File has `CRLF` line endings, so patch must have mixed `LF` and `CRLF` line endings. `core.autocrlf` of git broke my edits.
Now it is fixed.